app/controllers/account_reset/send_notifications_controller.rb

```diff
@@ -0,0 +1,22 @@
+module AccountReset
+  class SendNotificationsController < ApplicationController
+    before_action :authorize
+
+    def update
+      count = AccountResetService.grant_tokens_and_send_notifications
+      analytics.track_event(Analytics::ACCOUNT_RESET, event: :notifications, count: count)
+      render plain: 'ok'
+    end
+
+    private
+
+    def authorize
+      return if auth_token == Figaro.env.account_reset_auth_token
+      head :unauthorized
+    end
+
+    def auth_token
+      request.headers['X-ACCOUNT-RESET-AUTH-TOKEN']
```

Comment:
This looks like an api endpoint? I'm uncomfortable with us having a unique special header for authentication that's different for every api endpoint we end up creating. It's a lot to keep track of. Can we settle on a single header that we can use across all of our webhooks?

I'd also prefer that api endpoints (those not designed for user interaction directly) be under the `/api/` namespace, at least in the routes file.

Reply:
How about a generic X-API-AUTH-TOKEN ?